packages/cli/src/lib/cmds/build.ts

```diff
@@ -137,5 +139,31 @@ export async function build(
   );
 
   log.info(`Signed Android App generated at "${outputFile}"`);
+
+  try {
```

Comment:
I'd say you probably want to move this into its own function, that way we can make the `if` on line 153 a guard clause and return early afterwards.

Reply:
+1. I thought about it when writing the if statement but ended up forgetting. 🤦